index.html

```diff
@@ -160,25 +182,27 @@ <h2 class="title">About<br />Myself</h2>
             </ul>
             <button class="btn">Get my resume</button>
           </div>
-          <ul class="about-language">
-            <li>Languages</li>
-            <li><i class="fa-solid fa-chevron-down fa-xs"></i></li>
-            <ul>
-              <li class="language"><img src="images/Javascript.png" alt="Javascript" />Javascript</li>
-              <li class="language"><img src="images/HTML.png" alt="HTML" />HTML</li>
-              <li class="language"><img src="images/CSS.png" alt="CSS" />CSS</li>
+          <div class="skills">
+            <ul class="about-language">
+              <li class="arrow-down main">Languages</li>
+              <div class="program-language">
+                <ul>
+                  <li class="language"><img src="images/Javascript.png" alt="Javascript" />Javascript</li>
+                  <li class="language"><img src="images/HTML.png" alt="HTML" />HTML</li>
+                  <li class="language"><img src="images/CSS.png" alt="CSS" />CSS</li>
+                </ul>
+              </div>
+              <li class="arrow main">Frameworks</li>
+              <hr />
+              <li class="arrow main">Skills</li>
+              <hr />
             </ul>
-            <li>Frameworks</li>
-            <li><i class="fa-solid fa-chevron-right fa-xs"></i></li>
-            <hr />
-            <li>Skills</li>
-            <li><i class="fa-solid fa-chevron-right fa-xs"></i></li>
-            <hr />
-          </ul>
+          </div>
         </div>
       </section>
     </main>
-    <footer class="contact">
+    <!-- Fotter Section -->
+    <footer id="contact" class="contact">
       <div class="container container-padding-x">
         <h2 class="contact-title">Contact me</h2>
         <p class="contact-text">
```

Comment:
- [ ] Good job for this job you did :+1: , however , make sure the contact `section` fill all the width of the screen in the `desktop` version as it is represented in the Figma design. Kindly remove the space in `Red`. :100: on the picture below

Your design 

![Screenshot from 2022-12-08 22-59-31](https://user-images.githubusercontent.com/80458199/206574268-48c071c4-cfd4-42aa-bee4-1357310d5e88.png)